### DISCUSSION TITLE
Tower Defense - Software Architecture Doc

### DISCUSSION THREAD

**Discussion by TimWaeckerle** [Freaky2164/TowerDefense#25] [General]
Hi all,

For this week, we implemented some basic function. Our turrets are now facing the enemy while shooting. 

![ezgif-2-e36c8abe5f](https://user-images.githubusercontent.com/97874901/206243922-53f3f706-dfcb-4100-ba6d-151f77cf23ff.gif)

We also created our [Software Architecture Documentation](https://github.com/argastle/TowerDefense/blob/main/Projektmanagement/Software%20Architecture%20Documentation/Software%20Architecture%20Documentation.md)
Feel free to have a look.

Greetings,
TD Team

**Comment by momueller2** [comment]
Hi TD Team,
your progress on the game looks very promising, especially on the design front :).
I also like your focus on availabilty for all devices as its very similar to our project and I think for a game this should be a high priority.
- Brainstorm Team
